src/zenml/client.py

```diff
@@ -1205,6 +1209,234 @@ def register_pipeline(
         )
         raise AlreadyExistsException(error_msg)
 
+    def export_pipeline_runs(self, filename: str) -> None:
+        """Export all pipeline runs to a YAML file.
+
+        Args:
+            filename: The filename to export the pipeline runs to.
+
+        Raises:
+            RuntimeError: If no pipeline runs exist.
+        """
+        import json
+
+        from zenml.utils.yaml_utils import write_yaml
+
+        pipeline_runs = self.zen_store.list_runs(
+            project_name_or_id=self.active_project.id
+        )
+        if not pipeline_runs:
+            raise RuntimeError("No pipeline runs found.")
+        yaml_data = []
+        for pipeline_run in pipeline_runs:
+            pipeline_run.status = self.zen_store.get_run(pipeline_run.id).status
+            run_dict = json.loads(pipeline_run.json())
+            run_dict["steps"] = []
+            steps = self.zen_store.list_run_steps(run_id=pipeline_run.id)
+            for step in steps:
+                step.status = self.zen_store.get_run_step_status(step.id)
```

Comment:
I think you changed this for pipeline runs to be `get_run(...).status`. Does it make sense to have the same API for steps as well, `get_step(...).status`?

Reply:
Yes, good point, I think we could remove the `get_run_status()` method as well. Will do.